### PR TITLE
Add UI to edit users

### LINKED ISSUE #47: Add UI to edit users
- References: Issue #35

### PR DESCRIPTION
**Pull request by liannivin**
Status:
Required

Implementation Details:
Add UI to edit users.

close #47

### CODE REVIEW

File: react-time-track/src/services/user.js [hunk 1]
```diff
@@ -57,4 +58,41 @@ async function getUserProjects(user_id) {
   return response.json();
 }
 
-export { userList, createUser, getUserProjects };
+async function getUser(userId) {
+  console.log(userId);
```
Comment (diegotc86) [inline react-time-track/src/services/user.js]: Hi, don't forget to delete this console-logs

File: react-time-track/src/views/edit-user.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    React.useEffect(()=> {
+			console.log(user_id)
+      getUser(user_id)
+      .then(data => {setUser(data)
+    	});
+		}, []);
+
+	React.useEffect(()=> {
+		console.log(user);
```
Comment (diegotc86) [inline react-time-track/src/views/edit-user.js]: Same for this whole useEffect, it is used only for the console log, rigth?

File: react-time-track/src/views/edit-user.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	}, [user]);
+
+	function handleChange(e, key) {
+		setUser({...user, [key]: e.target.value})
```
Comment (diegotc86) [inline react-time-track/src/views/edit-user.js]: Hi, I think this is great. Could you explain me the benefits of it?

Comment (liannivin) [reply react-time-track/src/views/edit-user.js]: Sure. `handleChange` function use `e` and `key` as parameters. Firstly, I passed the `user` object properties using a `spread operator`, so I can access to its properties. Then, I pass `[key]` & `e.target.value` to hear all the changes in its `keys` and its `values` in order to replace them with the new data.

File: react-time-track/src/views/edit-user.js (hunk 2 above)
Comment (liannivin) [reply react-time-track/src/views/edit-user.js]: Thanks for notice them. I'll replace it right now.

File: react-time-track/src/services/user.js (hunk 1 above)
Comment (liannivin) [reply react-time-track/src/services/user.js]: Thanks for notice them. I'll replace it right now.

**Pull request merged**